show/main.py

```diff
@@ -827,6 +827,21 @@ def ip():
     """Show IP (IPv4) commands"""
     pass
 
+#
+# 'interfaces' subcommand ("show ip interfaces")
+#
+
+@ip.command()
+@click.argument('interfaces', required=False)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def interfaces(interfaces, verbose):
```

Comment:
The command `show ip interfaces` already exists (it is the next command in this file, just below the `get_if_admin_state()` and `get_if_oper_state()` helper functions--line 898 in your modified file). 

You're welcome to extend the existing command, but this PR is literally creating a second implementation of `show ip interfaces`.

Reply:
@batmancn: Have you seen my comment above? I suggest you close this PR and open a new one if you would like to extend the functionality of the existing `show ip interfaces` command.